lib/resources/r4/encounter_patch.yaml

```diff
@@ -0,0 +1,75 @@
+---
+name: Encounter
+field_name_base_url: http://hl7.org/fhir/R4/encounter-definitions.html#Encounter
+operations:
+  - name: replace-period-start
+    path: /period/start
+    operation: replace
+    cardinality: 0..1
+    type: dateTime
+    url: http://hl7.org/fhir/datatypes-definitions.html#Period.start
+    description: The start time of the encounter.
+    example: |
+      [
+      	{
+      		"op": "replace",
```

Comment:
fix up the indentation

Reply:
Fixed. https://github.com/cerner/fhir.cerner.com/commit/b3be566e03125de4bdc242726080efeaaef55e65#diff-97d84edc87e609cdbf80d6ad20cbc681R14